### PR TITLE
feat: Enable less-loader in cra

### PR DESCRIPTION
**Pull request by OshotOkill**
Less configuration could be a little bit complex since there're some differences between `development` mode and `production` mode.

I've tested it could work in simple scenarios. I hope someone can give feedbacks about their use cases such as [antd](https://ant.design/) or else.

### CODE REVIEW

**Comment by OshotOkill** [comment]
BTW to enable it we have to install `less` and `less-loader`:

```
yarn add less
yarn add --dev less-loader
```

**Comment by FezVrasta** [comment]
It'd be useful to document this in the README

**Comment by OshotOkill** [comment]
@FezVrasta Sure, I'll do that later.

File: readme.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ The functions documented below can be imported by name, and used in your `config
 
 ### addBabelPlugin(plugin)
 
-Adds a babel plugin. Whatever you pass for `plugin` will be added to Babel's `plugins` array. Consult their docs for more info.
```
Comment (FezVrasta) [inline readme.md]: Trailing spaces are meaningful in markdown

Comment (OshotOkill) [reply readme.md]: Sorry, my editor trims them automatically and I don't find trailing spaces anywhere else. I 'll restore them by another commit.

Comment (FezVrasta) [reply readme.md]: FIY 2 trailing spaces in markdown mean "newline", while an empty line means "new paragraph"

Comment (OshotOkill) [reply readme.md]: Yeah I've never used two trailing spaces (<br />) before.

**Comment by arackaf** [comment]
Thanks for adding the docs! Sorry, I’m on vacation in NYC this week, so this slipped through the cracks, but I’ll get this merged and published tomorrow.

**Pull request merged**